### PR TITLE
flake8 now automated

### PR DESCRIPTION
**Pull request by Marx314**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 deps = -r{toxinidir}/test-requirements.txt
 commands =
     nosetests --tests tests
+
+[testenv:flake8]
+commands = flake8
+
+[flake8]
+show-source = True
+max-line-length = 160
```
Comment (mbelang) [inline tox.ini]: 160? bit long no?

**Comment by fguillot** [comment]
👍

File: tox.ini (hunk 1 above)
Comment (Marx314) [inline tox.ini]: ./almanach/core/model.py:72:121: E501 line too long (131 > 120 characters)
    def **init**(self, entity_id, project_id, start, end, volume_type, size, last_event, name, attached_to=None, entity_type=TYPE):
                                                                                                                        ^
./tests/messages.py:34:121: E501 line too long (143 > 120 characters)
        "launched_at": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 16, 30, 02, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:55:121: E501 line too long (143 > 120 characters)
        "launched_at": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 16, 30, 02, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:56:121: E501 line too long (121 > 120 characters)
        "terminated_at": deletion_timestamp if deletion_timestamp else datetime(2014, 2, 18, 12, 5, 23, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:73:121: E501 line too long (143 > 120 characters)
        "launched_at": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:106:121: E501 line too long (143 > 120 characters)
        "launched_at": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:107:121: E501 line too long (141 > 120 characters)
        "timestamp": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:121:121: E501 line too long (123 > 120 characters)
        "timestamp": timestamp + timedelta(seconds=1) if timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:135:121: E501 line too long (123 > 120 characters)
        "timestamp": timestamp + timedelta(seconds=1) if timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:150:121: E501 line too long (143 > 120 characters)
        "launched_at": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:167:121: E501 line too long (143 > 120 characters)
        "launched_at": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:184:121: E501 line too long (143 > 120 characters)
        "launched_at": creation_timestamp + timedelta(seconds=1) if creation_timestamp else datetime(2014, 2, 14, 17, 18, 40, tzinfo=pytz.utc),
                                                                                                                        ^
./tests/messages.py:198:121: E501 line too long (135 > 120 characters)
                          deleted_at=None, terminated_at=None, state=None, os_type=None, os_distro=None, os_version=None, metadata={}):
                                                                                                                        ^
./tests/messages.py:275:121: E501 line too long (133 > 120 characters)
                                 volume_size=None, timestamp=None, created_at=None, launched_at=None, status=None, attached_to=None):
                                                                                                                        ^
./tests/adapters/test_database_adapter.py:169:121: E501 line too long (125 > 120 characters)
                                                             0).with_end(2014, 1, 1, 8, 0, 0).with_project_id("project_id")),
                                                                                                                        ^
./tests/adapters/test_database_adapter.py:177:121: E501 line too long (132 > 120 characters)
                                                                 0, 0).with_end(2014, 1, 1, 1, 0, 0).with_project_id("project_id")),
                                                                                                                        ^
./tests/adapters/test_database_adapter.py:179:121: E501 line too long (132 > 120 characters)
                                                                0, 0).with_end(2014, 1, 1, 11, 0, 0).with_project_id("project_id")),
                                                                                                                        ^
./tests/core/test_controller.py:366:121: E501 line too long (123 > 120 characters)
                                      some_volume.volume_type, some_volume.size, some_volume.name, some_volume.attached_to)

Comment (mbelang) [inline tox.ini]: Shouldn't we fix that instead of boosting the max-line-length line?

Comment (Marx314) [inline tox.ini]: to what value? 120? I tough 160 was reasonable but I'm flexible enough to have a shorter line if you have a specific number in you're head

**Comment by mbelang** [comment]
:+1:

**Pull request merged**